src/v2/view-builder/views/DeviceEnrollmentTerminalView.js

```diff
@@ -0,0 +1,57 @@
+import { loc } from 'okta';
+import BaseHeader from '../internals/BaseHeader';
+import HeaderBeacon from '../components/HeaderBeacon';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import OdaOktaVerifyTerminalView from '../components/OdaOktaVerifyTerminalView';
+import MdmOktaVerifyTerminalView from '../components/MdmOktaVerifyTerminalView';
+import Enums from 'util/Enums';
+
+const Header = BaseHeader.extend({
+  HeaderBeacon: HeaderBeacon.extend({
+    getBeaconClassName: () => 'mfa-okta-verify',
```

Comment:
- better use `AuthenticatorUtil#getIconClassNameForBeacon`.
- there is another place to hard code `mfa-okta-verify` and may want to refactor as well.

cc @jhe-okta

Reply:
> better use AuthenticatorUtil#getIconClassNameForBeacon.

if use `AuthenticatorUtil#getIconClassNameForBeacon` we need to pass `authenticatorType` from appState, right? But as I see, we don't have it in appState. So what is the decision? Where we should get it?


> there is another place to hard code mfa-okta-verify and may want to refactor as well.

What place?

Reply:
`AuthenticatorUtil#getIconClassNameForBeacon('app');`

Reply:
added `getIconClassNameForBeacon` to `getBeaconClassName` function